addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -323,21 +322,29 @@
                 </div>
             </div>
         </div>
-        <div class="card-footer bg-white text-600 px-3">
-            <div class="d-flex justify-content-between align-items-center">
-                <small t-if="channel.total_time" class="fw-bold" t-out="channel.total_time" t-options="{'widget': 'duration', 'unit': 'hour', 'round': 'minute'}"/>
-                <div class="d-flex flex-grow-1 justify-content-end">
-                    <t t-if="channel.is_member and channel.completed">
-                        <span class="badge text-bg-success pull-right"><i class="fa fa-check"/> Completed</span>
-                    </t>
-                    <div t-elif="channel.is_member and channel.channel_type != 'documentation'" class="progress w-50" style="height: 6px">
+        <div class="card-footer d-flex justify-content-between align-items-center">
+            <t t-if="0 &lt; channel.completion &lt; 100">
+                <div class="d-flex align-items-center gap-2 flex-basis-50">
+                    <div class="progress flex-grow-1" style="height: 6px">
                         <div class="progress-bar" role="progressbar" t-att-aria-valuenow="channel.completion" aria-valuemin="0" aria-valuemax="100" t-attf-style="width:#{channel.completion}%;" aria-label="Progress bar"/>
                     </div>
-                    <small t-else="" class="fw-bold"><t t-out="channel.total_slides"/> steps</small>
+                    <small><t t-out="channel.completion"/>%</small>
                 </div>
-            </div>
+                <button class="btn btn-sm btn-primary">Continue</button>
+            </t>
+            <t t-elif="channel.is_member and channel.completion == 0">
+                <small t-if="channel.total_time" t-out="channel.total_time" t-options="{'widget': 'duration', 'unit': 'hour', 'round': 'minute', 'format': 'narrow'}"/>
+                <button class="btn btn-sm btn-primary">Start now</button>
+            </t>
+            <t t-elif="channel.is_member and channel.completed">
+                <span class="badge text-bg-success"><i class="fa fa-check" aria-hidden="true"/> Completed</span>
+            </t>
```

Comment:
These can be merged by adding `t-if` to the `span` (unless it's overridden somewhere else)?
It wasn't the case before but we can be more efficient.